src/js/components/Scope/DatasetScopePicker.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { List, Avatar, Space, Typography } from 'antd';
+import { useTranslationCustom, useTranslationDefault } from '@/hooks';
+import { Link, useLocation } from 'react-router-dom';
+import { FaDatabase } from 'react-icons/fa';
+import { getCurrentPage } from '@/utils/router';
+import type { Project } from '@/types/metadata';
+
+type DatasetScopePickerProps = {
+  parentProject: Project;
+  isSingleProject?: boolean;
+};
+export const DatasetScopePicker = ({ parentProject, isSingleProject }: DatasetScopePickerProps) => {
+  const td = useTranslationDefault();
+  const t = useTranslationCustom();
+  const location = useLocation();
+  const baseURL = '/' + location.pathname.split('/')[1];
+  const page = getCurrentPage();
+  return (
+    <Space direction="vertical" style={{ display: 'flex' }}>
+      <Space align="baseline" size="large">
+        <Typography.Title level={4} className="no-margin-top">
+          {td('Project')}: {t(parentProject.title)}
+        </Typography.Title>
+        {isSingleProject ? (
+          // give clear scope option in single project mode
+          <Link to={baseURL}>
+            <Typography.Link>{td('Clear')}</Typography.Link>
+          </Link>
+        ) : (
+          <Link to={`${baseURL}/p/${parentProject.identifier}/${page}`} key="3">
+            <Typography.Link>{td('Select')}</Typography.Link>
+          </Link>
+        )}
+      </Space>
+      <Typography.Text>{t(parentProject.description)}</Typography.Text>
+      <Typography.Title level={5} className="no-margin-top">
+        {td('Datasets')}
```

Comment:
there should either be a clear button here to clear the selected dataset ("Clear selection" – *only if* a dataset is selected), or above the Select button should be re-enabled for single-project scopes with different wording, *only if* a dataset is selected

Reply:
Clear dataset button is now only present when the parentProject includes the selected dataset scope.